ibm/mas_devops/roles/cp4d_db2wh/defaults/cpd40/main.yml

```diff
@@ -0,0 +1,11 @@
+# A temporary workaround for a bug in CP4D DB2, details in story 1372 & 1472, to be removed once the bug is fixed in CP4D (4.x version)
+apply_db2u_sts_patch: false
+db2wh_load_from_config: false
```

Comment:
unless it's been reimplemented `db2wh_load_from_config` was removed from use in the master branch, it's the only way it works now, and this is unnecessarily added back I think as part of a bad merge?

Reply:
Actually while Alex was making the changes for the rest api to work for cpd 3.5, I was working in parallel with the cpd 4.0 and used the db2ucluster approach as base, so to make it work both ways, had to reintroduce this flag back, as db2ucluster approach does not use config map to store db2 info. It's going to take up more time to understand the api changes for v4 to implement same approach as we did for v3.5 db2wh, so maybe this could be an enhancement for near future? Let me know your thoughts.

Reply:
remove the need of `db2wh_load_from_config` and now if `cpd_version == 'cpd40'` then we don't load the config map, if `cpd_version == 'cpd35'` then we load the config map as rest api expects.